stable/datadog/templates/cluster-agent-pdb.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.clusterAgent.podDisruptionBudget -}}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ template "datadog.fullname" . }}-cluster-agent
+  labels:
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    app.kubernetes.io/name: "{{ template "datadog.fullname" . }}"
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "datadog.fullname" . }}-cluster-agent
+{{ toYaml .Values.clusterAgent.podDisruptionBudget | indent 2 }}
+{{- end -}}
```

Comment:
I think we can directly provide directly the proper `spec`
In the case of the cluster-agent what is important it to have always at least one instance.

```yaml
spec:
  minAvailable: 1
  selector:
    matchLabels:
      app: {{ template "datadog.fullname" . }}-cluster-agent
```

so maybe instead of have the spec provided by `.Values.clusterAgent.podDisruptionBudget` we can imagine to have only a boolean: `.Values.clusterAgent.createPodDisruptionBudget`.
WDYT?